lib/ts/recipe/multitenancy/recipe.ts

```diff
@@ -142,12 +142,7 @@ export default class Recipe extends RecipeModule {
         throw new Error("should never come here");
     };
 
```

Comment:
remove from error.ts and also search in the whole repo for these error types, and keep the error file with BAD_INPUT like in other recipes